.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,17 @@
+# Ref: https://help.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser
+blank_issues_enabled: true
+contact_links:
+- name: 🙋  Usage Questions
+  url: https://github.com/osl-incubator/cookiecutter-python/discussions
+  about: |
+    Use Cookiecutter's GitHub Discussions to ask "How do I do X with Cookiecutter?".
+- name: 📖  Tutorial
+  url: https://github.com/osl-incubator/cookiecutter-python
+  about: |
+    The Cookiecutter tutorial is continually updated and provides an in depth walkthrough
+    of how to use the latest release of Cookiecutter.
+- name: 📝  Cookiecutter Code of Conduct
+  url: https://github.com/osl-incubator/cookiecutter-python/blob/main/CODE_OF_CONDUCT.md
+  about: Expectations for how people will interact with each other on Cookiecutter's GitHub.
+
+Thanks [Pyhf](https://github.com/scikit-hep/pyhf/tree/main/.github/ISSUE_TEMPLATE) for  for coming up with such detailed and great templates slightly_smiling_face
```

Comment:
Where will this be visible?

Reply:
I think this will not be visible. Removing the credit from this file.